common/src/thrift/file_system_master.thrift

```diff
@@ -74,60 +74,53 @@ service FileSystemMasterClientService extends common.TachyonService {
   /**
    * Marks a file as completed.
    */
-  void completeFile( /** the id of the file */ 1: i64 fileId,
+  void completeFile( /** the path of the file */ 1: string path,
       /** the method options */ 2: CompleteFileTOptions options)
     throws (1: exception.TachyonTException e)
 
   /**
-   * Creates a file and returns the id of the file.
+   * Creates a directory
    */
-  i64 create( /** the path of the file */ 1: string path,
-      /** the options for completing the file */ 2: CreateTOptions options)
+  void createDirectory( /** the path of the directory */ 1: string path,
+      /** the method options */ 2: CreateDirectoryTOptions options)
     throws (1: exception.TachyonTException e, 2: exception.ThriftIOException ioe)
 
   /**
-   * Frees the given file from Tachyon and returns whether the free operation succeeded.
+   * Creates a file
```

Comment:
I understand that you wanted to merge this to avoid keep having to resolving merge conflicts, but please fix this and similar unanswered nits in a follow up PR, thanks